packages/components/src/theme/ThemeProvider.tsx

```diff
@@ -36,12 +36,14 @@ export interface ThemeProviderProps {
    * tell the provider to activate the base themes.
    */
   themes: ThemeData[] | null;
+  waitForActivation?: boolean;
```

Comment:
As mentioned in my other comment - do we really need this prop on ThemeProvider? If we do, add a comment about it and what the component does if this is true.